config/install/core.entity_view_display.node.localgov_publication_page.default.yml

```diff
@@ -24,10 +24,18 @@ content:
     third_party_settings: {  }
     weight: 0
     region: content
+  field_localgov_services_landing:
```

Comment:
This shouldn't have the field_ prefix. Just localgov. See here: https://github.com/localgovdrupal/localgov/wiki/Naming-conventions#set-field-prefix-to-localgov

Reply:
Ach - looks like this is the name of the field in https://github.com/localgovdrupal/localgov_services/blob/2.x/modules/localgov_services_landing/config/install/field.storage.node.field_localgov_services_landing.yml

Sorry, should have spotted that!